packages/core/admin/_internal/node/core/plugins.ts

```diff
@@ -86,7 +89,9 @@ const getEnabledPlugins = async ({
          * User plugin paths are resolved from the entry point
          * of the app, because that's how you import them.
          */
-        pathToPlugin: userPluginConfig.resolve,
+        pathToPlugin: convertSystemPathToModulePath(
```

Comment:
should we make this consistent with the other loader if we are really cleaning this up ? 

path / modulePath

Reply:
that's a nice idea.